docs/user-guide/global-configurations/cluster-and-environments.md

```diff
@@ -1,87 +1,81 @@
 # Cluster And Environments
 
-The Global configuration provides a feature of `Cluster & Environment` in which you can add your Kubernetes clusters and environment.
+You can add your existing Kubernetes clusters and environments here.
 
-Select the Cluster & Environment section of global configuration and click on `Add Cluster` to add your cluster.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/cluster-and-environments.png)
 
 ## Add Cluster:
 
 To add a cluster on devtron, you must have superadmin access.
 
-Navigate to the `Global Configurations` → `Clusters and Environments` on devtron and click on `Add Cluster`. Provide the below informations to add your kubernetes cluster:
+Navigate to the `Global Configurations` → `Clusters and Environments` on devtron and click on `Add Cluster`.
 
-1. Name
-2. Kubernetes Cluster Info
-   * Server URL
-   * Bearer token
-3. Prometheus Info
-   * Prometheus endpoint
-     * Basic
-       * Username
-       * Password
-     * Anonymous
-   * TLS Key
-   * TLS Certificate
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/add-clusters.png)
 
-
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/cluster-and-environments/cluster-1.jpg)
+ Provide the informations mentioned below to add your kubernetes cluster:
```

Comment:
Provide the below information to add your kubernetes cluster:

Reply:
done